src/components/formControl/index.tsx

```diff
@@ -23,22 +28,35 @@ const FormControl: FC<PropsWithChildren<Props>> = ({
   id,
   label,
   hint,
+  tooltip,
 }) => {
   const isInvalid = !!errorMessage;
 
+  const renderLabel = (
+    <FormLabel htmlFor={id}>
+      {label}
+      {isRequired ? <>&nbsp;</> : null}
+    </FormLabel>
+  );
+
+  const renderLabelWithTooltip = (
+    <Flex justifyContent="start">
```

Comment:
Stack might be more speaking in this case https://chakra-ui.com/docs/components/stack/usage#notes-on-stack-vs-flex